projects/netgrif-components/src/lib/authentication/session-idle/session-idle.component.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, Input} from '@angular/core';
+import {
+    AbstractSessionIdleComponent,
+    ConfigurationService,
+    LoggerService,
+    SessionIdleTimerService,
+    UserService
+} from '@netgrif/components-core';
+import {MatDialog} from "@angular/material/dialog";
+import {Router} from "@angular/router";
+import {PopupSessionIdleComponent} from "./popup-session-idle/popup-session-idle.component";
+
+@Component({
+    selector: 'nc-session-idle',
+    templateUrl: './session-idle.component.html',
+    styleUrls: []
+})
+export class SessionIdleComponent extends AbstractSessionIdleComponent {
+
+    @Input() alertAt? = 45;
+
+
+    constructor(protected dialog: MatDialog,
```

Comment:
don't we have our own wrapper around the MatDialog service?

Reply:
Nope